src/webview/src/components/summary/dataProvider.js

```diff
@@ -0,0 +1,243 @@
+            Aggregated: ({ value }) => `${value} Effects`,
+          },
+          {
+            Header: 'Category',
+            accessor: 'category',
+            aggregate: 'count',
+            Aggregated: ({ value }) => `${value} Categories`,
+          },
+          {
+            Header: 'Root Cause',
+            accessor: 'rc',
+            disableGroupBy: true,
+            aggregate: 'count',
+            Aggregated: ({ value }) => `${value} Root Causes`,
+          },
+        ],
+      },
+      {
+        Header: 'Properties',
+        columns: [
+          {
+            Header: 'Status',
+            accessor: 'value',
+            Filter: SelectColumnFilter,
+            filter: 'includes',
+            aggregate: getStatusAggregator,
+            Aggregated: ({ value }) => {
+              return (
+                <React.Fragment>
+                  <Chip
+                    size="small"
+                    icon={<CheckBoxOutlineBlankIcon />}
+                    color="primary"
+                    label={value[0]}
+                    variant="outlined"
+                  />
+                  <Chip
+                    size="small"
+                    icon={<CheckBoxIcon />}
+                    color="primary"
+                    label={value[1]}
+                    variant="outlined"
+                  />
+                  <Chip
+                    size="small"
+                    icon={<ErrorIcon />}
+                    color="secondary"
+                    label={value[2]}
+                    variant="outlined"
+                  />
+                </React.Fragment>
+              );
+              // return (
+              //   <React.Fragment>
+              //     <Badge badgeContent={value[0]} max={99} color="primary" overlap="circle" showZero>
+              //       <CheckBoxOutlineBlankIcon />
+              //     </Badge>
+              //     <Badge badgeContent={value[1]} max={99} color="primary" overlap="circle" showZero>
+              //       <CheckBoxIcon color="primary" />
+              //     </Badge>
+              //     <Badge badgeContent={value[2]} max={99} color="secondary" overlap="circle" showZero>
+              //       <ErrorIcon color="secondary" />
+              //     </Badge>
+              //   </React.Fragment>
+
+              // );
+            }
+
+          },
+          {
+            Header: 'Background',
+            accessor: 'background',
+            filter: 'customFilter',
+            disableGroupBy: true,
+          },
+          {
+            Header: 'Instructions',
+            accessor: 'instructions',
+            filter: 'customFilter',
+            disableGroupBy: true,
+          },
+          {
+            Header: 'Comments',
+            accessor: 'comments',
+            filter: 'customFilter',
+            disableGroupBy: true,
+          },
+        ],
+      },
+    ],
+    []
+  )
+}
+
+var FbPathChangeHook = undefined;
```

Comment:
could be inside the function "SummaryDataProvider"?

Reply:
I agree.